.travis.yml

```diff
@@ -13,6 +13,7 @@ env:
   - LAYER=gmp PHP="72 73 74"
   - LAYER=memcached PHP="72 73 74"
   - LAYER=xdebug PHP="72 73 74"
+  - LAYER=pgsql PHP="72 73 74"
```

Comment:
If you want to please my OCD you can move this line up one too =)

Reply:
Also fixed this.